package.json

```diff
@@ -70,6 +68,7 @@
     "@types/webpack-env": "^1.16.0",
     "@typescript-eslint/parser": "^4.26.0",
     "concurrently": "^6.2.0",
+    "dayjs": "^1.11.7",
```

Comment:
Why did this get moved? (dayjs itself is tiny, but I'd rather just have one instance of it so we can all use one set of plugins).

Reply:
We had issues where `dayjs` was not to be installed for development with `yarn` Moving it to the dev dependencies kinda solved that.

Reply:
It's fine to have it in both `devDependencies` and `peerDependencies`... that's our usual pattern. However, it definitely should be in `peerDepedencies` as otherwise we'll have runtime issues.

Reply:
Note that in the monorepos, the usual pattern is actually that it's in `peerDependencies` for the app and `devDependencies` for the repo packages.json, but for single-app packages, it will all be in the one place.